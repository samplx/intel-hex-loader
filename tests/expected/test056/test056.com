?Vp+^q+r+s!F4!UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU