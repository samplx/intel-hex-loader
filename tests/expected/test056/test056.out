initial memoryLimit  = 0x0100
program size setting = none
loading COM file     < 'tests/com-files/256-55.com'
bytes loaded         = 256
loading HEX file     < 'tests/hex-files/sixteen-bytes-with-3F-at-0100.hex'
X
loaded.
final memoryLimit    = 0x01ff
file size in bytes   = 256
dumping memory to    > 'test056.com'
